data/sql/updates/pending_db_world/remove-bop-quest-loot.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM reference_loot_template WHERE (`Entry` = 4110) AND (`Item` = 30520); --remove row with Gold-Trimmed Cuffs
+DELETE FROM reference_loot_template WHERE (`Entry` = 4110) AND (`Item` = 30986); --remove row with Bloodforged Guard
```

Comment:
```suggestion
DELETE FROM reference_loot_template WHERE (`Entry` = 4110) AND (`Item` IN (30520, 30986));
```

Reply:
yeah makes sense. better to add them together. just still a little bit afraid to mess things up lol. I just did a course in websecurity and sql injections were a part of it ahaha